lib/hardware_services/inventory_drivers/MockInventoryDriver.py

```diff
@@ -42,4 +42,16 @@ def list_clouds(self, quadsinstance):
     def list_hosts(self, quadsinstance):
         print "listing hosts"
 
+    def load_data(self, quadsinstance, force):
+        print("data is loaded")
+
+    def write_data(self, quadsinstance, doexit = True):
+        print("data is written")
+
+    def sync_state(self, quadsinstance):
+        print("data is synchronized")
+
+    def init_data(self, quadsinstance, force):
+        print("data is initialized")
```

Comment:
@vsemp can you just change the print statements to python 2.7 format? Besides that, i think this looks great! We can merge as soon as you fix the prints!